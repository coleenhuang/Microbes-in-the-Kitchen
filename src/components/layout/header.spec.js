import React from "react"
import renderer from "react-test-renderer"

import Header from "./header"

describe("Header", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Header siteTitle="Microbes in the Kitchen" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})