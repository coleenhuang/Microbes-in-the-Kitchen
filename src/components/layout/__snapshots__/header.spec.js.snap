// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header
  style={
    Object {
      "display": "block",
      "marginBottom": "1rem",
      "marginTop": "1.45rem",
    }
  }
>
  <div
    style={
      Object {
        "margin": "0 auto",
        "maxWidth": 960,
        "padding": "1.45rem 1.0875rem",
      }
    }
  >
    <h1
      style={
        Object {
          "margin": 0,
          "textAlign": "center",
        }
      }
    >
      <a
        aria-current="page"
        className=""
        href="/"
        onClick={[Function]}
        onMouseEnter={[Function]}
        style={
          Object {
            "color": "black",
            "fontFamily": "Ovo",
            "textDecoration": "none",
          }
        }
      >
        siteMetadata.title
      </a>
    </h1>
  </div>
</header>
`;
